### PR TITLE
testing/py-graphviz: upgrade to 0.8.3

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: testing/py-graphviz/APKBUILD [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-pkgver=0.8.1
+_pkgname=${pkgname#py-}
+pkgver=0.8.3
 pkgrel=0
 pkgdesc="Simple interface for Graphviz"
 url="https://github.com/xflr6/graphviz"
 arch="noarch"
 license="MIT"
-depends=""
+options="!check" # 0.8.3 enable when py-atomicwrites is added to pytest depends
```
Comment (nmeum) [inline testing/py-graphviz/APKBUILD]: This should have been fixed by 783ed893ce9b6cb8d3bc076f610bd259a6b18294

**Comment by TBK** [comment]
@nmeum The check will not run because it is missing a dependency. I have made a PR for the dependency https://github.com/alpinelinux/aports/pull/4655

**Comment by nmeum** [comment]
> @nmeum The check will not run because it is missing a dependency. I have made a PR for the dependency #4655

In that case I would prefer merging #4655 before merging this one.

**Comment by andypost** [comment]
0.8.4 released

File: testing/py-graphviz/APKBUILD [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-pkgver=0.8.1
+_pkgname=${pkgname#py-}
+pkgver=0.8.3
 pkgrel=0
 pkgdesc="Simple interface for Graphviz"
 url="https://github.com/xflr6/graphviz"
 arch="noarch"
 license="MIT"
-depends=""
+options="!check" # Depends on pytest-cov
```
Comment (nmeum) [inline testing/py-graphviz/APKBUILD]: #4655 has been merged. Can check be enabled now?

**Comment by andypost** [comment]
Version 0.9 is out https://github.com/xflr6/graphviz/blob/master/CHANGES.txt

File: testing/py-graphviz/APKBUILD (hunk 2 above)
Comment (andypost) [reply testing/py-graphviz/APKBUILD]: Now it needs https://pypi.org/project/pytest-mock/

Comment (andypost) [reply testing/py-graphviz/APKBUILD]: with a package from https://github.com/alpinelinux/aports/pull/5310 version 0.9 passes tests locally

**Comment by andypost** [comment]
0.10.1 is out

**Pull request closed without merging**